src/test/java/resources/batch.feature

```diff
@@ -172,3 +177,18 @@ Feature: Batch Details API test
     Then I am able to parse batch summary response
     And Status of transaction is "COMPLETED"
     And I should have matching total txn count and successful txn count in response
+
+  @gov @batch-teardown
+  Scenario: BD-013 Batch aggregate API Test
+    Given I have the demo csv file "test.csv"
+    And I have tenant as "rhino"
+    And I generate clientCorrelationId
+    And I have private key
+    And I generate signature
+    When I call the batch transactions endpoint with expected status of 202
+    Then I should get non empty response
+    And I am able to parse batch transactions response
+    And I fetch batch ID from batch transaction API's response
+    Then I will sleep for 5000 millisecond
+    When I call the batch aggregate API with expected status of 200
+    Then I should get non empty response
```

Comment:
The test case should have 2-3 fields checked, expected count/ success percentage etc.
non empty response is not enough